broker-daemon/orderbook/index.js

```diff
@@ -164,6 +164,32 @@ class Orderbook {
     return trades
   }
 
+  /**
+   * Get orders in the orderbook for a given side up to a given limit. If no limit is provided, gets all orders
+   * @param {String} side Side of the orderbook to get orders for (i.e. `BID` or `ASK`)
+   * @param {String} limit int64 String of the the amount of orders to return.
+   * @return {Array<MarketEventOrder>} A promise that resolves MarketEventOrders for the limited records
+   */
+  getOrders ({ side, limit }) {
```

Comment:
It seems like this is very similar to `all()`. Is there a way that we can remove `all()` and replace it with this new function? Where else is `all()` used?